docs/src/guide/input_output.jl

```diff
@@ -0,0 +1,83 @@
+# # Input and output formats
+#md # [![](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/guide/@__NAME__.ipynb)
+#md # [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/guide/@__NAME__.ipynb)
+
+# This section provides an overview of the input and output formats
+# supported by DFTK, usually via integration with a third-party library.
+#
+# ## Reading input formats supported by ASE
+# ASE is short for the
+# [atomistic simulation environment](https://wiki.fysik.dtu.dk/ase/index.html),
+# a Python package to simplify the process of setting up, running and
+# analysing results from atomistic simulations across different programs.
+# If it is installed it is automatically used by DFTK in order to read a wide range
+# of input files, such as xyz, CIF, input files of various other codes
+# (e.g. Quantum Espresso, VASP, ABINIT, CASTEP, ...).
+# The full list of formats
+# can be found in the [ASE IO documentation](https://wiki.fysik.dtu.dk/ase/ase/io/io.html).
+#
+# As an example we start the calculation of a simple antiferromagnetic iron crystal
+# using a Quantum-Espresso input file, [Fe_afm.pwi](Fe_afm.pwi).
+# From this file the lattice, atomic positions and the initial magnetisation are read.
+# For more details about calculations on magnetic systems
+# using collinear spin, see [Collinear spin and magnetic systems](@ref).
+
+using DFTK
+
+## Parse the Quantum Espresso input file
+qe_input = "Fe_afm.pwi"
+atoms    = load_atoms(qe_input)
```

Comment:
say somewhere that ASE does the magic of auto-detecting it's a QE input file (but it could have been some other format)